### ISSUE TITLE
x-editable should not contain source code of 3rd part libraries

### ISSUE THREAD

**Issue by simivar** [vitalets/x-editable#885]
I had a weird error today with DatePicker not working corrently and after some time I found out that you've used same datepicker in x-editable and you just copied whole source code into x-editable source code. That's not how it should be done - because of that I always got error about date not being correct - your datepicker overwritten my datepicker from own file and in your code it's in super-old version 1.3. 

Please do something about it.

**Comment by lorvent** [comment]
i guess thats only for demonstration purpose.

if you use bower then those packages will be downloaded directly from their repo.

**Comment by simivar** [comment]
Actually it is in lib when you download the latest release from github.

**Comment by arturmamedov** [comment]
No, it yet in main js file https://github.com/vitalets/x-editable/blob/develop/dist/bootstrap-editable/js/bootstrap-editable.js#L4902
It not the right way
